Maelstrom/State.cs

```diff
@@ -51,6 +51,7 @@ public class State
         public bool GfAbilitiesBasics { get; set; } = false;
         public bool GfAbilitiesIncludeItemOnly { get; set; } = false;
         public bool GfAbilitiesSwapSets { get; set; } = false;
+        public String GfAbilitiesLimit { get; set; } = "21";
```

Comment:
You can make this an int & the converter on the combobox will handle parsing & stuff

Reply:
Awesome. I'm pretty new to UI development in .NET, so any pointers/critiques you have are more than welcome.